### ISSUE TITLE
Md file extension 

### ISSUE THREAD

**Issue by sisyphosloughs** [nextcloud/notes#257]

**Comment by sisyphosloughs** [comment]
I really do love this app. Markdown is great for editing notes. Bevore I got to know this App I used Markdown for my Notes anyway. So simply copied my Markdown files into the notes folder, and the App accetped that. It also keept the .md extension, when I edited my old files with app. This also worked with Android app. Very nice!

The only thing is that the app stores notes with .txt extension. This is verry unhandy, if I want to edit notes with an editor which supports Markdown. All of the editors treat  files with .txt extension as plain Ascii files. So I need to be explained, that this file is a Markdown. Thats quite sad, because both the web App and the Android App are able to deal with the .md files.

So in short: could you please integrate a an option to customize the way how the app stores its file extension?

**Comment by korelstar** [comment]
Such an option is already available since version 2.4.0 of the Web app (see #223). You can use the settings area (bottom left) in order to change the default suffix for new notes.

**Comment by korelstar** [comment]
duplicate of #24

**Comment by korelstar** [comment]
Duplicate of #24
